2024-12-12 19:58:31,198 : WARNING : Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\Users\Admin\PycharmProjects\pythonProjectUniversity\Модуль 12\tests_12_4.py", line 71, in test_run
    test = Runner(5)
           ^^^^^^^^^
  File "C:\Users\Admin\PycharmProjects\pythonProjectUniversity\Модуль 12\tests_12_4.py", line 13, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
2024-12-12 19:58:31,199 : WARNING : Неверная скорость для Runner
Traceback (most recent call last):
  File "C:\Users\Admin\PycharmProjects\pythonProjectUniversity\Модуль 12\tests_12_4.py", line 61, in test_walk
    test = Runner('Вася', -10)
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\PycharmProjects\pythonProjectUniversity\Модуль 12\tests_12_4.py", line 18, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -10
